lib/apiservers/engine/backends/container.go

```diff
@@ -271,15 +273,19 @@ func (c *Container) ContainerExecCreate(name string, config *types.ExecConfig) (
 	if err != nil {
 		return "", InternalServerError(err.Error())
 	}
+
+	// This does not appear to be working... we may need to do this check further down the stack. it is going to be very race filled.
```

Comment:
ah that was just an investigative comment :) I thought it was removed sorry :P I was thinking that it was an early check on state that was far from the source of truth. But I figured it is more of a soft check anyways in case the user tried to exec an already off container.

Reply:
I am going to remove this completely. It is a good check to have and we can always handle things down stream as well.